pennylane/interfaces/jax.py

```diff
@@ -454,11 +457,10 @@ def execute_wrapper(params):
     @execute_wrapper.defjvp
     def execute_wrapper_jvp(primals, tangents):
         """Primals[0] are parameters as Jax tracers and tangents[0] is a list of tangent vectors as Jax tracers."""
-        new_tapes = [_copy_tape(t, a) for t, a in zip(tapes, primals[0])]
-
         if isinstance(gradient_fn, qml.gradients.gradient_transform):
+            at_max_diff = _n == max_diff
             _args = (
-                new_tapes,
+                set_parameters_on_copy_and_unwrap(tapes, primals[0], unwrap=at_max_diff),
```

Comment:
Minor: set it outside the function just for readability.

Reply:
moved outside of tuple